src/index.html

```diff
@@ -4,9 +4,362 @@
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="#home" class="header__logo">
+          <img src="images/logo.png" alt="logo">
+        </a>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a href="#apple" class="is-active nav__link">apple</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#samsung" class="nav__link">samsung</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#smartphones" class="nav__link">smartphones</a>
+            </li>
+
+            <li class="nav__item">
+              <a
+                href="#laptops&computers"
+                class="nav__link"
+                data-qa="nav-hover"
+              >
+                laptops & computers
+              </a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#gadgets" class="nav__link">gadgets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#tablets" class="nav__link">tablets</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#photo" class="nav__link">photo</a>
+            </li>
+
+            <li class="nav__item">
+              <a href="#video" class="nav__link">video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <main class="main">
+      <article class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+          class="card__image"
+        >
+
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <h3 class="card__code">Product code: 195434</h3>
+
+        <div class="card__rate">
+          <div class="card__stars stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <h4 class="card__price card__price--title">Price:</h4>
+          <p class="card__price card__price--value">$2,199</p>
+        </div>
+
+        <div class="card__buy">
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </article>
+      <article class="card">
```

Comment:
don't forget about empty line between siblings
```suggestion
      </article>
      
      <article class="card">
```